.github/workflows/CI.yml

```diff
@@ -82,6 +82,38 @@ jobs:
           cargo run zigbuild --manifest-path tests/bindgen-exhaustive/Cargo.toml --target aarch64-unknown-linux-gnu
           cargo run zigbuild --manifest-path tests/bindgen-exhaustive/Cargo.toml --target x86_64-pc-windows-gnu
           cargo run zigbuild --manifest-path tests/bindgen-exhaustive/Cargo.toml --target aarch64-apple-darwin
+      - name: Test RUSTFLAGS
+        shell: bash
+        run: |
+          set -e
+          cargo install x86-instruction-set-analyzer --version 0.1.0
```

Comment:
Could you release binaries for this project so we don't need to compile it every time on GitHub Actions?

Reply:
Thanks! Really good project. Now x86-instruction-set-analyzer is binary installed in CI.